templates/commands.conf.j2

```diff
@@ -475,3 +475,15 @@ object CheckCommand "cst_pgsql" {
   vars.my_port = "$port$"
   vars.my_login = "$credentials$"
 }
+
+object CheckCommand "systemd" {
+  import "plugin-check-command"
+  command = [PluginDir + "/check_systemd_service.sh"]
```

Comment:
You don't actually parse the arguments, so `--value` is a little bit misleading here. I guess we can keep it this way.